### ISSUE TITLE
Can you add 'month' tracking to report?

### ISSUE THREAD

**Issue by nikivdev** [jason0x43/alfred-toggl#38]
I would like to have the time shown for the current month as well as today, yesterday and week. This would be really useful for me.

I hope it would be possible to add. Thank you.

**Comment by jason0x43** [comment]
This is possible, but it involves a bit more work when downloading info from Toggl because their default download range (I believe) is just the last 10 days.

**Comment by nikivdev** [comment]
Yes, I suppose I can wait to get the data shown. It would be a neat feature though.

**Comment by stacyk** [comment]
I would like a report of last month :)
